src/internal/clusterstate/v2.4.0.go

```diff
@@ -24,6 +26,25 @@ var state_2_4_0 migrations.State = state_2_3_0.
 			return errors.Wrap(err, "could not create default project")
 		}
 		return nil
+	}).
+	Apply("Grant ProjectCreator to all users at cluster level", func(ctx context.Context, env migrations.Env) error {
+		key := auth_server.ResourceKey(&auth.Resource{Type: auth.ResourceType_CLUSTER})
+		rb := &auth.RoleBinding{Entries: make(map[string]*auth.Roles)}
+		if err := auth_server.RoleBindingCollection(nil, nil).ReadWrite(env.Tx).Get(key, rb); err != nil {
```

Comment:
we should move src/server/auth/server/authdb.go ->  src/internal/pfsdb/authdb.go so that the migration package doesn't depend on the server package